src/stellar_format.c

```diff
@@ -1059,30 +1147,130 @@ static void format_set_trust_line_flags(tx_context_t *txCtx) {
     push_to_formatter_stack(&format_set_trust_line_trustor);
 }
 
-static const format_function_t formatters[] = {
-    &format_create_account,
-    &format_payment,
-    &format_path_payment_strict_receive,
-    &format_manage_sell_offer,
-    &format_create_passive_sell_offer,
-    &format_set_options,
-    &format_change_trust,
-    &format_allow_trust,
-    &format_account_merge,
-    &format_inflation,
-    &format_manage_data,
-    &format_bump_sequence,
-    &format_manage_buy_offer,
-    &format_path_payment_strict_send,
-    &format_create_claimable_balance,
-    &format_claim_claimable_balance,
-    &format_begin_sponsoring_future_reserves,
-    &format_end_sponsoring_future_reserves,
-    &format_revoke_sponsorship,
-    &format_clawback,
-    &format_clawback_claimable_balance,
-    &format_set_trust_line_flags,
-};
+static void format_liquidity_pool_deposit_max_price(tx_context_t *txCtx) {
+    strcpy(detailCaption, "Max Price");
+    uint64_t price =
+        ((uint64_t) txCtx->txDetails.opDetails.liquidityPoolDepositOp.maxPrice.n * 10000000) /
+        txCtx->txDetails.opDetails.liquidityPoolDepositOp.maxPrice.d;
+    print_amount(price, NULL, txCtx->network, detailValue, DETAIL_VALUE_MAX_SIZE);
+    push_to_formatter_stack(&format_operation_source);
+}
+
+static void format_liquidity_pool_deposit_min_price(tx_context_t *txCtx) {
+    strcpy(detailCaption, "Min Price");
+    uint64_t price =
+        ((uint64_t) txCtx->txDetails.opDetails.liquidityPoolDepositOp.minPrice.n * 10000000) /
+        txCtx->txDetails.opDetails.liquidityPoolDepositOp.minPrice.d;
+    print_amount(price, NULL, txCtx->network, detailValue, DETAIL_VALUE_MAX_SIZE);
+    push_to_formatter_stack(&format_liquidity_pool_deposit_max_price);
+}
+
+static void format_liquidity_pool_deposit_max_amount_b(tx_context_t *txCtx) {
+    strcpy(detailCaption, "Max Amount B");
+    print_amount(txCtx->txDetails.opDetails.liquidityPoolDepositOp.maxAmountB,
+                 NULL,
+                 txCtx->network,
+                 detailValue,
+                 DETAIL_VALUE_MAX_SIZE);
+    push_to_formatter_stack(&format_liquidity_pool_deposit_min_price);
+}
+
+static void format_liquidity_pool_deposit_max_amount_a(tx_context_t *txCtx) {
+    strcpy(detailCaption, "Max Amount A");
+    print_amount(txCtx->txDetails.opDetails.liquidityPoolDepositOp.maxAmountA,
+                 NULL,
+                 txCtx->network,
+                 detailValue,
+                 DETAIL_VALUE_MAX_SIZE);
+    push_to_formatter_stack(&format_liquidity_pool_deposit_max_amount_b);
+}
+
+static void format_liquidity_pool_deposit_liquidity_pool_id(tx_context_t *txCtx) {
+    strcpy(detailCaption, "Liquidity Pool ID");
+    print_binary_summary(
```

Comment:
From what I understand, liquidity pool IDs are a very sensitive field.
I suggest to display the whole value of this field, using `print_binary` instead for `print_binary_summary`.

(Same apply to all the liquidity pool IDs in `format_liquidity_pool_withdraw_liquidity_pool_id` etc.)